test/unit_tests/test_eegneuralnet.py

```diff
@@ -52,6 +52,29 @@ def __init__(
     def forward(self, x):
         return self.preds
 
+class MockModule2(EEGModuleMixin, torch.nn.Module):
+    def __init__(
+            self,
+            n_outputs=None,
+            n_chans=None,
+            chs_info=None,
+            n_times=None,
+            input_window_seconds=None,
+            sfreq=None,
+    ):
+        super().__init__(
+            n_outputs=n_outputs,
+            n_chans=n_chans,
+            chs_info=chs_info,
+            n_times=n_times,
+            input_window_seconds=input_window_seconds,
+            sfreq=sfreq,
+        )
+        self.final_layer = torch.nn.Conv1d(self.n_chans, self.n_outputs, self.n_times)
+
+    def forward(self, x):
```

Comment:
proposition to inherit from `MockModule1` and only overridde `forward` behavior to actually use `self.final_layer`

Reply:
ok 👍